plenum/server/replica.py

```diff
@@ -2318,17 +2334,17 @@ def dequeue_prepares(self, viewNo: int, ppSeqNo: int):
             while self.preparesWaitingForPrePrepare[key]:
                 prepare, sender = self.preparesWaitingForPrePrepare[
                     key].popleft()
-                self.logger.info("{} popping stashed PREPARE{}".format(self, key))
+                # self.logger.info("{} popping stashed PREPARE{}".format(self, key))
                 self.process_three_phase_msg(prepare, sender)
                 i += 1
             self.preparesWaitingForPrePrepare.pop(key)
-            self.logger.info("{} processed {} PREPAREs waiting for PRE-PREPARE for"
-                             " view no {} and seq no {}".format(self, i, viewNo, ppSeqNo))
+            # self.logger.info("{} processed {} PREPAREs waiting for PRE-PREPARE for"
```

Comment:
Please leave it on INFO level (do not comment out)

Reply:
ok